sdk/communication/communication-sms/samples/v1/javascript/package.json

```diff
@@ -1,32 +1,32 @@
 {
   "name": "azure-communication-sms-samples-js",
   "private": true,
-  "version": "0.1.0",
-  "description": "Azure Communication Service SMS client library samples for JavaScript",
+  "version": "1.0.0",
+  "description": "Azure Communication Services client library samples for JavaScript",
   "engine": {
-    "node": ">=8.0.0"
+    "node": ">=12.0.0"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Azure/azure-sdk-for-js.git"
+    "url": "git+https://github.com/Azure/azure-sdk-for-js.git",
+    "directory": "sdk/communication/communication-sms"
   },
   "keywords": [
+    "azure",
+    "cloud",
     "Azure",
-    "Communication",
-    "SMS",
-    "Node.js",
-    "JavaScript"
+    "communication",
+    "sms"
```

Comment:
was the removal  of `javascript` and. `node.js` intentional?

Reply:
this is autogenerated but i can check where it's coming from